integreat_cms/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -70,9 +70,12 @@ <h1 class="heading">
                     <th class="text-sm text-left uppercase py-3 pr-2">
                         {% translate "Marked as read by" %}
                     </th>
-                    <th class="text-sm text-right uppercase py-3 pr-2">
+                    <th class="text-sm text-left uppercase py-3 pr-2">
                         {% translate "Date" %}
                     </th>
+                    <th class="text-sm text-left uppercase py-3 pr-2">
```

Comment:
I think this still should be `text-right` as we had before, but I'm fine either way